utils.go

```diff
@@ -402,34 +402,25 @@ func sanitizeRootfs(id, rootfs string) string {
 	return rootfs
 }
 
-// getLinuxHeaderMounts returns a list of read-only mounts of the host's linux kernel headers.
-func getLinuxHeaderMounts() ([]specs.Mount, error) {
+// getLinuxHeaderMounts returns a list of read-only mounts of the host's linux
+// kernel headers.
+func getLinuxHeaderMounts(kernelHdrPath string) ([]specs.Mount, error) {
 
-	kernelRel, err := sysbox.GetKernelRelease()
-	if err != nil {
-		return nil, err
-	}
-
-	kernelHdr := "linux-headers-" + kernelRel
-
-	path := filepath.Join("/usr/src/", kernelHdr)
-	if _, err := os.Stat(path); err != nil {
-		return nil, err
-	}
-
-	mounts := []specs.Mount{}
+	var src = kernelHdrPath
```

Comment:
nit: no need for the "src" variable.

Reply:
I know. Just trying to avoid another 100+ column-wide line further below.

Reply:
Let's keep kernelHdrPath, it's more descriptive than `src`.

Reply:
i don't agree. "src" definition is 3 lines above so there's no room for confusion, and it enhances readability by keeping all code within decent column margins.